content/en/blog/2020/wasmhub-istio

```diff
@@ -0,0 +1,51 @@
+
+We’ve had a lot of interest in our innovation in this area, and we’ve been working hard to further the capabilities of WebAssembly Hub and the workflows it supports. Today we are thrilled to announce new enhancements to WebAssembly Hub that evolve the viability of WebAssembly+Envoy for production, improve the developer experience, and streamline using Wasm with Envoy in Istio. 
+
+## Evolving toward production
+The Envoy community is working hard to bring Wasm support into the upstream project (right now it lives on a working development fork), with Istio declaring Wasm support an Alpha feature. In [Gloo 1.0 we also announced](https://www.solo.io/blog/announcing-gloo-1-0-a-production-ready-envoy-based-api-gateway/) early, non-production support for Wasm. 
+
+Although it’s still maturing in each individual project, there are things we can do right now to improve the foundation for production support. 
+
+The first area we are targeting is standardizing what a WebAssembly extension for Envoy looks like. Together with Google and the Istio community, we’ve defined an open specification for bundling and distributing WebAssembly modules as OCI images. This specification provides a powerful model for distributing any type of Wasm module including Envoy extensions. 
+
+You can join the effort here: https://github.com/solo-io/wasm-image-spec
+
+We’re also improving the experience of deploying Wasm extensions into an Envoy-based framework running in production. In the Kubernetes ecosystem, it is considered best practice in production to use declarative CRD-based configuration to manage cluster configuration. The new [WebAssembly Hub Operator](https://docs.solo.io/web-assembly-hub/latest/tutorial_code/wasme_operator/) adds a single, declarative CRD which automatically deploys and configures Wasm filters to Envoy proxies running inside of a Kubernetes cluster. This operator enables GitOps workflows and cluster automation to manage Wasm filters without human intervention or imperative workflows.
+
+Lastly, the interactions between developers of Wasm extensions and the teams that deploy them need some kind of role-based access, organization management, and facilities to share, discover, and consume these extensions. The WebAssembly Hub adds team management features like permissions, organizations, user management, sharing and more. 
+
+## Improve developer experience
```

Comment:
s/Improve/Improved/

Reply:
to match the tense of the other heading, I updated to "Improving the developer experience"